service/worker/deletenamespace/fx.go

```diff
@@ -44,8 +45,18 @@ import (
 )
 
 type (
+	activityWorkerConfig struct {
+		// worker concurrency option changes require a restart to take effect
+		// default to 0 to use SDK-defined defaults
```

Comment:
small nit: could we make those two lines two distinct sentences? without capitalization or a period, I read this as one sentence and it confused me.